File: submissions/sm_009_chinmay/week_03/day_3/session_2/count_consonants.js
// Print the count of all the consonants in the given string

function countOfConsonants(string) {
 var consonantsCount = string.length
 // console.log(consonantsCount)
 string =  string.toLowerCase()
 for(i=0; i<string.length; i++){

  if(string.charAt(i)== "a"){
   consonantsCount -= 1
  }
  else if(string.charAt(i)=="e"){
   consonantsCount -= 1
  }
  else if(string.charAt(i)=="i"){
   consonantsCount -= 1
  }
  else if(string.charAt(i)=="o"){
   consonantsCount -= 1
  }
  else if(string.charAt(i)=="u"){
   consonantsCount -= 1
  }

 }
 console.log(consonantsCount)
}

countOfConsonants("Consonants")

File: submissions/sm_102_amit/week_13/day_4/session_1/count_consonants.py
# Print the count of all the consonants in the given list of strings
strList = ["apple", "banana", "cherry", "orange", "kiwi", "melon", "mango"]
vowel = ["a", "e", "i", "o", "u"]
count = 0
for string in strList:
    for x in string:
        if x in vowel:
            continue
        else:
            count += 1
    print(string, count)
    count = 0
